packager/react-packager/src/DependencyResolver/Package.js

```diff
@@ -92,9 +92,22 @@ class Package {
 }
 
 function getReplacements(pkg) {
-  return pkg['react-native'] == null
-    ? pkg.browser
-    : pkg['react-native'];
+  let rn = pkg['react-native']
+  let browser = pkg.browser
+  if (rn == null) return browser
```

Comment:
Don't use short version of `if`...use curly braces and put the return on a new line.